Modules/Module_Processing/Module_Population/xASL_stat_GetROIstatistics.m

```diff
@@ -538,7 +538,7 @@
 			DataIm(:) = NaN;
 		end
 		%% 4.d Show ROIs projected on ASL image (%% PM %% : ALSO DONE BELOW)
-		LabelIM = xASL_vis_Convert4D_3D_atlas(xASL_im_Column2IM(SubjectSpecificMasks(:,[1:3:end]), x.S.masks.WBmask));
+		LabelIM = xASL_vis_Convert4D_3D_atlas(xASL_im_Column2IM(SubjectSpecificMasks(:, 1:3:end), x.S.masks.WBmask));

```

Comment:
This will create a conflict with another recent issue, so because it doesn't seem necessary, I would revert this one. But your decision, you will probably be the one resolving the conflict :)

Reply:
Haha - no worries - that's a conflict I can handle :)